README.md

```diff
@@ -1,48 +1,54 @@
-# Bio-Formats
+# OME Common C++
 
-[![Build Status](https://travis-ci.org/openmicroscopy/bioformats.png)](http://travis-ci.org/openmicroscopy/bioformats)
```

Comment:
In the case of this repo, travis testing is possible.  But in general it's going to be quite difficult--running the superbuild to build all the components is simply not possible with the time constraints.  Since this repo has no dependencies, it's easy to do.  But for the other downstream repos, this will be difficult; even if we download the cached builds, we have no guarantee they are compatible with currently open PRs--which will be the case when we have changes spanning multiple repos.  I'm unsure that Travis will be viable for testing.